source/MainPanel.cpp

```diff
@@ -189,7 +192,8 @@ void MainPanel::Draw()
 // The planet panel calls this when it closes.
 void MainPanel::OnCallback()
 {
-	engine.Place();
+	if(player.TeleportationStatus() != PlayerInfo::MOVED)
```

Comment:
You tested consecutive landing and non landing teleport missions, right?

Reply:
I will have teleport being limited to only work if the player is landed

Reply:
Good
But I mostly meant did you check for a mission that teleports you to another planet where there is another teleport mission that would also happen (but does it trigger if its a `landing` mission?

Reply:
If the player gets teleported to a planet, they trigger all landing missions

Reply:
K all good for me then